openapiSpec.yaml

```diff
@@ -0,0 +1,828 @@
+                default:
+                    description: Unexpected error
+        put:
+            operationId: updateSolution
+            tags:
+                - Solution
+            summary: update a solution with the specified id
+            parameters:
+                -   name: id
+                    in: path
+                    required: true
+                    description: the id of the specified solution
+                    schema:
+                        type: string
+                        format: guid
+            requestBody:
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/Solution'
+            responses:
+                '200':
+                    description: Solution created succesfully
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/Solution'
+                '400':
+                    description: Not a valid request
+                '404':
+                    description: Solution with the specified id not found
+                default:
+                    description: Unexpected error
+        delete:
+            operationId: deleteSolution
+            tags: 
+                - Solution
+            summary: delete a solution with the specified id
+            parameters:
+                -   name: id
+                    in: path
+                    required: true
+                    description: the id of the specified solution
+                    schema:
+                        type: string
+                        format: guid
+            responses:
+                '204':
+                    description: Solution deleted succesfully
+                '400':
+                    description: Not a valid request
+                '404':
+                    description: Solution with the specified id not found
+                default:
+                    description: Unexpected error
+        patch:
+            operationId: updateCount
+            tags:
+                - Solution
+            summary: count view count up by one
+            parameters:
+                -   name: id
+                    in: path
+                    required: true
+                    description: the id of the specified solution
+                    schema:
+                        type: string
+                        format: guid
+            responses:
+                '204':
+                    description: Solution patched succesfully
+                '400':
+                    description: Not a valid request
+                '404':
+                    description: Solution with the specified id not found
+                default:
+                    description: Unexpected error
+    /solutions/{id}/likes/{userId}:
```

Comment:
Waar worden de solution likes opgeslagen?

Reply:
ik neem aan dat ze een aparte tabel in de solution service hebben die deze relatie bijhoudt

Reply:
ik zat trouwens ook een boolean veld in het solution object maken die ijo=houdt of de huidige gebruiker de solution heeft geliked